VariablesHomework/VariablesHomework.cs

```diff
@@ -161,18 +169,18 @@ public void Casting()
             // what am i doing here that make the above work? (Add a comment with your answer)
             double d = 15.5;
             int c = (int)d;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass by ONLY filling in the answer
+            Assert.IsTrue(c == 15.50); // Make this pass by ONLY filling in the answer
```

Comment:
What happens to a floating-point (float, double, or decimal) value when we cast it to an integral type?